DashAI/back/api/api_v1/endpoints/experiments.py

```diff
@@ -67,6 +75,61 @@ async def get_experiment(experiment_id: int, db: Session = Depends(get_db)):
     return experiment
 
 
+@router.post("/validation")
+async def validate_columns(
+    params: ColumnsValidationParams,
+    db: Session = Depends(get_db),
```

Comment:
After the dependency injection merge, we need to change how we manage some dependencies, such as
```suggestion
    session_factory: Callable[..., ContextManager[Session]] = Depends(
        Provide[Container.db.provided.session]
```
Then call it using
```
with session_factory() as db:
        try:
            dataset = db.get(Dataset,...)            
 ```

Reply:
This is not needed for this PR. Just for the merge to develop